README.md

```diff
@@ -10,9 +10,9 @@ LORIS is a web-accessible database solution for neuroimaging, providing a secure
  * LINUX (supported on Ubuntu 14.04 and CentOS 6.5) or Mac OS X (tested for Mavericks - OS X 10.9)
  * Apache2 (libapache2-mod-php5)
  * MySQL (libmysqlclient15-dev mysql-client mysql-server)
- * PHP/PEAR 5.3+ (php5 php-pear php5-mysql php5-gd)
+ * PHP 5.3+ (php5 php5-mysql php5-gd php5-sqlite)
+ * PEAR (php-pear)
```

Comment:
Are we? I still see 'pear' listed under 'repositories' in our composer.json
It's also throughout our codebase like our install script, dockerfile, etc so if we are, we can send a separate pull request to get rid of all pear stuff. I don't want to mix that too much into this pull request.